src/test/java/utilities/Util.java

```diff
@@ -15,4 +19,25 @@ public static void wait(int seconds) {
 		}
 	}
 
+	/*
+	 * Switches the driver focus to the child window
+	 * 
+	 * @Author: Erlyn Gammad
+	 */
+
+	public static void switchToSeparateWindow(WebDriver driver) {
```

Comment:
1, What this function does, is basically `switchingToTheSecondAvailableWindow`
2, JavaDoc should be right above the function without empty line
3, Fix formatting